pom.xml

```diff
@@ -644,6 +644,11 @@
         </exclusions>
       </dependency>
       <dependency>
+        <groupId>io.jsonwebtoken</groupId>
+        <artifactId>jjwt</artifactId>
+        <version>0.6.0</version>
```

Comment:
be careful, jjwt brings the dependencies `jackson-databind 2.4.2` and `jackson-annotations 2.4.0`, where Elasticsearch brings `jackson-core 2.6.6`. I suggest that:
* all these dependencies are explicitly defined in our poms
* we use a single version: 2.6.6